.gitignore

```diff
@@ -2,6 +2,7 @@
 *.iml
 target
 bundles-dist
+.DS_Store
```

Comment:
Why we need it?

Reply:
https://en.wikipedia.org/wiki/.DS_Store.   unfortunately this is generated by MAC file system while accessing these files, so for good sake and be safe I added this file to be ignored, which is not needed to be committed